src/listeners/commands/_chat-input-shared.ts

```diff
@@ -0,0 +1,18 @@
+import { getSupportedUserLanguageT } from '#lib/i18n/translate';
+import { type ChatInputCommandErrorPayload } from '@sapphire/framework';
+import type { ChatInputSubcommandErrorPayload } from '@sapphire/plugin-subcommands';
+import { MessageFlags } from 'discord.js';
+import { flattenError, generateUnexpectedErrorMessage, resolveError } from './_shared.js';
+
+export async function handleCommandError(error: unknown, payload: ChatInputCommandErrorPayload | ChatInputSubcommandErrorPayload) {
+	const { interaction } = payload;
+	const t = getSupportedUserLanguageT(interaction);
+	const resolved = flattenError(payload.command, error);
+	const content = resolved ? resolveError(t, resolved) : generateUnexpectedErrorMessage(interaction.user.id, payload.command, t, error);
+
+	try {
+		if (interaction.replied) await interaction.followUp({ content, flags: MessageFlags.Ephemeral });
+		else if (interaction.deferred) await interaction.editReply({ content });
+		else await interaction.reply({ content, flags: MessageFlags.Ephemeral });
```

Comment:
```suggestion
		else await interaction.reply({ content, ephmemeral: true });
```
Why use flags and not just `ephmemeral: true`?

Reply:
ephemeral*, and used to because the HTTP bots, but I can update them.